Shamaz/ViewController.swift

```diff
@@ -8,72 +8,107 @@
 
 import UIKit
 
+@available(iOS 13.0, *)
```

Comment:
Won't it prevent your app from running on iOS 11?

Reply:
Yes, I was not sure how to setup iOS11. Is it really necessary?